TemplateApplication.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -203,8 +203,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/StanfordSpezi/SpeziOnboarding",
       "state" : {
-        "revision" : "e0b11e101a920fa01f92e0ebf77fa0bb1c0265ef",
-        "version" : "0.5.1"
+        "branch" : "feat/export-consent-forms",
+        "revision" : "9b9f362c95c807902a0b7f7bbf1ded238f84748b"
```

Comment:
Just a conversation to remind us that we have to remove that before merging the PR.